Source/ASCollectionView.mm

```diff
@@ -949,7 +955,8 @@ - (NSInteger)collectionView:(UICollectionView *)collectionView numberOfItemsInSe
   return [_dataController.visibleMap numberOfItemsInSection:section];
 }
 
-- (CGSize)collectionView:(UICollectionView *)collectionView layout:(UICollectionViewLayout *)collectionViewLayout sizeForItemAtIndexPath:(NSIndexPath *)indexPath
+- (CGSize)collectionView:(UICollectionView *)collectionView
+                  layout:(UICollectionViewLayout *)layout sizeForItemAtIndexPath:(NSIndexPath *)indexPath
```

Comment:
Since you changed the parameter name from `collectionViewLayout` to `layout`, you need to update this [line](https://github.com/TextureGroup/Texture/pull/440/files#diff-8b82b1556d1293f6f9789beb236c9271R964).